catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/IdentityNodeInitialization.java

```diff
@@ -68,18 +72,27 @@
 
     private SlotTypeHelper slotTypeHelper = new SlotTypeHelper();
 
+    private ScheduledExecutorService executorService;
+
     private InternationalStringTypeHelper internationalStringTypeHelper =
             new InternationalStringTypeHelper();
 
-    public void init() throws PrivilegedActionException {
-        Security.runAsAdminWithException(() -> {
-            if (!federationAdminService.getLocalRegistryIdentityMetacard()
-                    .isPresent()) {
-                createIdentityNode();
-            }
-            return null;
-        });
+    public void init() {
+        try {
+            Security.runAsAdminWithException(() -> {
+                if (!federationAdminService.getLocalRegistryIdentityMetacard().isPresent()) {
+                    createIdentityNode();
+                }
+                return null;
+            });
+        } catch (PrivilegedActionException e) {
+            LOGGER.warn("Error checking for local registry identity node. Will try again later");
+            executorService.schedule(this::init, RETRY_INTERVAL, TimeUnit.SECONDS);
```

Comment:
Is there any case where this will always fail and you'll keep looping through your `init` forever? 

I think there should there be a bailout option if you fail some, possibly configurable, number of times.